source/atlas-data-lake-testing/tests/runCommand.yml

```diff
@@ -0,0 +1,19 @@
+database_name: &database_name "test"
```

Comment:
`database_name` is no longer used in this file. You can either add `database_name: *database_name` to the APM assertion below for delete this line.

Reply:
I have added the assertion.